Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class BalanceStats
+    {
+        private const string InValidOutput = "N/A.";
+        private StringBuilder _persons;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_persons' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)